common/lib/xmodule/xmodule/static_content.py

```diff
@@ -97,7 +97,7 @@ def _write_styles(selector, output_root, classes):
 
     module_styles_lines = []
     module_styles_lines.append("@import 'bourbon/bourbon';")
-    module_styles_lines.append("@import 'bourbon/addons/button';")
+    module_styles_lines.append("@import 'bourbon/bourbon-deprecated-upcoming';")
```

Comment:
NOTE: Since there is no more ``bourbon/addons/button`` partial/utility, I'm pointing this python controlled CSS concatenation/compilation to Bourbon's upcoming deprecated partial for now (its where the ``button()`` mixin was moved to until the next release).

My hope is that by the next Bourbon upgrade, we'll at least have atomic parts of the pattern library in place, such as buttons, and won't need this any more. Compilation warnings should be silent locally with this.